e2e/jest-puppeteer.config.js

```diff
@@ -5,7 +5,7 @@
 
 const fp = require('lodash/fp');
 const puppeteer = require('puppeteer');
-const isHeadless = process.env.HEADLESS !== 'false';
+const isHeadless = (process.env.HEADLESS || 'true') === 'true';
```

Comment:
so we're defaulting to Headless if this value is missing.  is that what we want?

Reply:
yes, that's the intention. it's like that since day 1. I changed code to make it explicitly clear.

Reply:
great!